app/src/main/java/com/concordia/mcga/helperClasses/DatabaseHelper.java

```diff
@@ -5,22 +5,22 @@
 import android.database.sqlite.SQLiteException;
 import android.database.sqlite.SQLiteOpenHelper;
 import android.util.Log;
-
+import com.concordia.mcga.activities.BuildConfig;
+import com.concordia.mcga.exceptions.MCGADatabaseException;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.sql.SQLException;
 
 
 //Part of the code was taken from https://blog.reigndesign.com/blog/using-your-own-sqlite-database-in-android-applications/
 public class DatabaseHelper extends SQLiteOpenHelper {
```

Comment:
All the factories that we will have in our project, that want to access the information found in the database will have to interact with this object. That's what i meant by God class. There is no way viable around this.